### DISCUSSION TITLE
404 Page is Loading first in Cpanel Hosted NextJs Website

### DISCUSSION THREAD

**Discussion by benjamin-paul-s** [vercel/next.js#26098] [Help]
I hosted a Next Js app in Cpanel using terminal. When I use my domain name to open the site it first goes to 404 page. If I click the link in 404 page created using ( <Link href="/" as={process.env.PUBLIC_URL + "/"}> ) then it starts working in the domain name itself. If I refresh the page it again shows 404 page. This didn't happen when I hosted in vercel.

.htaccess
RewriteEngine On
RewriteRule ^$ http://127.0.0.1:3000/ [P,L]
RewriteCond %{REQUEST_FILENAME} !-f
RewriteCond %{REQUEST_FILENAME} !-d
RewriteRule ^(.*)$ http://127.0.0.1:3000/$1 [P,L]

**Comment by sadiquekp** [comment]
[benjamin-paul-s](https://github.com/benjamin-paul-s) did you find the solution ?
